extension/content/utils/api.js

```diff
@@ -91,10 +99,29 @@ export default class NormandyAPI {
     });
   }
 
-  async fetchRecipe(id, { version = 3 }) {
+  async fetchRecipe(id) {
     return this.request({
       url: `recipe/${id}/`,
-      version,
+    });
+  }
+
+  async saveRecipe(id, data) {
+    let url = "recipe/";
```

Comment:
nit: should be `const`

Reply:
doesn't that make url not happy when it changes to "recipe/<id>" below?